tv_apps/smart-home/js/home.js

```diff
@@ -675,12 +675,15 @@
       if(scrollable === this.cardScrollable && this._folderCard &&
                         itemElem.dataset.cardId !== this._folderCard.cardId &&
                         !this.cardScrollable.isHovering) {
-        this._cleanFolderScrollable();
+        this.cleanFolderScrollable();
       }
     },
 
-    _cleanFolderScrollable: function() {
-      if (this._focusScrollable === this.folderScrollable) {
+    cleanFolderScrollable: function() {
+      var currentFocusedElement = this.spatialNavigator.getFocusedElement();
```

Comment:
can we add an argument called `silent` or `doNotChangeFocus` or something for this? Then we don't need to determine id anymore.